docs/man/man1/kubectl-exec.1

```diff
@@ -19,7 +19,7 @@ Execute a command in a container.
 .SH OPTIONS
 .PP
 \fB\-c\fP, \fB\-\-container\fP=""
-    Container name. If omitted, the first container in the pod will be chosen
+    Container name. If omitted, the first app container in the pod will be chosen
```

Comment:
But since you can exec init container properly, init container is also be the first container, not sure why you make this change here ...